### PR TITLE
[TF] Minor changes.

### PR DESCRIPTION
**Pull request by dan-zheng**

### CODE REVIEW

File: stdlib/public/TensorFlow/Ops.swift [hunk 1]
```diff
@@ -1501,26 +1501,27 @@ public extension Tensor {
...
+      let slice = Raw.stridedSlice(
+        self, begin: Tensor<Int32>([index]), end: Tensor<Int32>([index + 1]),
+        strides: Tensor<Int32>([1]))
+      return slice.squeezingShape(at: 0)
     }
     set {
-      let left = self[0..<index]
-      let right = self[index+1..<_TFGetScalarOrDie(shapeTensor[0].handle)]
-      self = Raw.concatV2([left, newValue.rankLifted(), right], axis: Tensor<Int32>(0))
+      let count = _TFGetScalarOrDie(shapeTensor[0].handle)
```
Comment (dan-zheng) [inline stdlib/public/TensorFlow/Ops.swift]: Would we like to change `_TFGetScalarOrDie(shapeTensor[0].handle)` to something else?
Perhaps `shape[0]` directly?

cc @rxwei @jekbradbury

Comment (rxwei) [reply stdlib/public/TensorFlow/Ops.swift]: Yes please!

Comment (dan-zheng) [reply stdlib/public/TensorFlow/Ops.swift]: Done. NFC status revoked.

**Comment by dan-zheng** [comment]
`test/TensorFlowRuntime/simple.swift` fails for me locally, not sure why.
Triggering CI regardless.

**Comment by dan-zheng** [comment]
@swift-ci Please test tensorflow

**Review by rxwei — APPROVED** [review]
LGTM, though there is no issue with the original code style. The benefit I can see is then change from `.shapeTensor` to `.shape`, which is more efficient without stdlib optimizations.

**Pull request merged**